.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "Dev",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": { "VARIANT": "18" }
+	},
+
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"dbaeumer.vscode-eslint",
+				"ms-vscode.powershell",
+				"eg2.vscode-npm-script",
+				"mikestead.dotenv",
+				"christian-kohler.npm-intellisense"
+			]
+		}
+	},
+	"forwardPorts": [1521],
```

Comment:
what's using 1521?

Reply:
Oracle when you run it inside the container